### DISCUSSION TITLE
Suggesting some features

### DISCUSSION THREAD

**Discussion by CristianMG92** [Chia-Network/chia-blockchain#3485] [Ideas]
Hi all! I started plotting about a week and a half ago and I was wondering if is it possible to add some "improvements" I think. I'm Cristian, UX/UI designer and marketing mannager in some car dealerships in Mallorca.

I started plotting in parallel but later I started to add plots to queue. In one day I can configure the next 3 days in all the computers. The real problem is when it suddenly shuts down, or the GUI stops working or something.

So... what about a delay of a queue? With that I mean, I configure a queue of 6 plots, but I want it to start in 45 min, so my first plot can arrive to 31%+/- and start the second queue. And do that some times.

And a second suggestion is to add an automatic description to the plot while it's plotting. With that could be:

- The hour that the plot starts (if there is a delay, calculate it).
- How many threads, plots, and ram (at least) is in the queue.
- If there is an error it could show it next to the %.

I have some more ideas, but I need to think about that. If something of that is possible and I can do something for help just tell me. Thanks guys.

**Comment by sargonas** [comment]
while not a native function, you might want to check out some of the community generated tools, like plotman. It should enable you to do something along the lines of what you’ve outlined here in the interim, until the devs include this functionality for us in the GUI
